kurtosis_package/shared_utils.star

```diff
@@ -1,10 +1,5 @@
-# NOTE: this is a temporary workaround due to foundry-rs not having arm64 images
-FOUNDRY_IMAGE = ImageBuildSpec(
-    image_name="Layr-Labs/foundry",
-    build_context_dir="./dockerfiles/",
-    build_file="contract_deployer.Dockerfile",
-    target_stage="foundry",
```

Comment:
We can simplify the `contract_deployer.Dockerfile` dockerfile too

Reply:
pushed, if this is what you meant